content/departments/sales/sales-enablement/running-batch-changes-server-side-beta/index.md

```diff
@@ -0,0 +1,57 @@
+# Running batch changes server-side is moving to beta
+
+## Quicklinks
+
+- Enablement session recording (TODO)
+- Demo video
+- [Release post](https://github.com/sourcegraph/about/pull/5471)
+
+## Summary
+
+[Running batch changes server-side](https://docs.sourcegraph.com/batch_changes/explanations/server_side#limitations) is moving to beta. This feature was initially built to enable large customers to run batch changes on very large amounts of repositories. It also comes with UX improvements that will allow developers to iterate on batch changes more quickly. This feature requires installing executors, which comes with additional resource requirements and maintenance. Executors are like CI agents, that Sourcegraph uses to offload expensive tasks. [Executors](https://docs.sourcegraph.com/admin/executors) also power code intelligence [auto-indexing](https://docs.sourcegraph.com/code_intelligence/explanations/auto_indexing). Running batch changes server-side is available both for self-hosted and managed instances customers, as long as customers are able to deploy and host the executors themselves. We do not manage executors on behalf of managed instance customers at this point.
+
+## What is running batch changes server-side?
+
+Batch Changes is a way to apply and track code changes to many repositories and codehosts. Batch Changes users develop a code rewrite script, and Batch Changes takes care of applying it on many repos, open changesets, and provide a tracking dashboard.
+
+A key step in this workflow today is that users run their batch change locally thanks to a local command line interface (`src-cli`). For large amounts of repositories or resource-intensive code rewrites, running `src-cli` locally could take an impractical amount of time and be brittle.
+
+![](local-run.svg)
```

Comment:
Would recommend using `<object>` over `<img>` for diagrams for accessibility (lets screen readers navigate through the text boxes in the diagram)
```suggestion
<object data="local-run.svg"></object>
```